cmd/config.go

```diff
@@ -303,6 +303,47 @@ func getServiceConfig(reader *bufio.Reader) (*v3.RKEConfigServices, error) {
 		return nil, err
 	}
 	servicesConfig.Kubelet.InfraContainerImage = infraPodImage
+
+	// Add the flexvolume mount to kublet ExtraBinds
+	flexVolMount, err := getConfig(reader, "Enable the FlexVolume driver mount", "no")
+	if err != nil {
+		return nil, err
+	}
+
+	if strings.ContainsAny(flexVolMount, "Yes y Y yes") {
+		gkeBind := "/home/kubernetes/flexvolume:/home/kubernetes/flexvolume"
+		rkeBind := "/var/lib/kubelet/volumeplugins:/var/lib/kubelet/volumeplugins"
+
+		getCE, err := getConfig(reader, "Deploying to GKE or RKE environment", "rke")
```

Comment:
"RKE environment" sounds a bit confusing - as it doesn't reflect the infrastructure provider (aws, baremetal, etc) it gets deployed to - especially when we oppose it to "GKE environment". Instead you can do something like:

a) `Is GKE cloud?` (false by default)
b) `Host type` (Node by default, GKE as an option). I'm trying to avoid use of `cloudProvider` construct as it's already being used for diff purpose.

Reply:
Would `"Is GKE your cloud provider?" [no]` be more appropriate? 

With the changes I mentioned above we could just do away with this.

Reply:
Thats better. Do you happen to know if other cloud providers might need the same logic as well? If no, `is GKE` (bool) should be fine. If yes, "enum" choice is more appropriate

Reply:
I was thinking of letting the user just input the appropriate path. One reason being is that I have seen users in slack wanting/needing to change the flexvolume path for the kubelet.

Reply:
```go
const 	defaultFlexVolumeBindPath = "/var/lib/kubelet/volumeplugins:/var/lib/kubelet/volumeplugins"
....
     if strings.ContainsAny(flexVolMount, yesAnswer) {
		// Allow the user to set custom flex volume mount points
		FlexVolumeBind, err = getConfig(reader, "Flex volume bind mount path for kubelet service", defaultFlexVolumeBindPath)

		if err != nil {
			return nil, err
		}
		//If a custom volume bind is passed, use it and set the --volume-plugin-dir extra_args for kubelet service
		if FlexVolumeBind != defaultFlexVolumeBindPath {
			volPluginDir := strings.Split(FlexVolumeBind, ":")
			
			servicesConfig.Kubelet.ExtraArgs["--volume-plugin-dir"] = volPluginDir[1]
		}
		
		servicesConfig.Kubelet.ExtraBinds = append(servicesConfig.Kubelet.ExtraBinds, FlexVolumeBind)

	}

```